[2025-02-21T17:07:47.925+0000] {processor.py:186} INFO - Started process (PID=18350) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:07:47.925+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-21T17:07:47.927+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:07:47.927+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:07:47.930+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:07:47.930+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-21T17:07:47.930+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:07:47.930+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-21T17:07:47.930+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:07:47.937+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2025-02-21T17:23:00.794+0000] {processor.py:186} INFO - Started process (PID=21768) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:23:00.796+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-21T17:23:00.798+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:23:00.798+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:23:00.801+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:23:00.801+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-21T17:23:00.802+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:23:00.802+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-21T17:23:00.802+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:23:00.811+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2025-02-21T17:23:58.662+0000] {processor.py:186} INFO - Started process (PID=22013) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:23:58.664+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-21T17:23:58.666+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:23:58.665+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:23:58.668+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:23:58.668+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-21T17:23:58.668+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:23:58.668+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-21T17:23:58.668+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:23:58.677+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.020 seconds
[2025-02-21T17:24:56.709+0000] {processor.py:186} INFO - Started process (PID=22291) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:24:56.711+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-21T17:24:56.713+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:24:56.713+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:24:56.715+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:24:56.715+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-21T17:24:56.716+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:24:56.715+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-21T17:24:56.716+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:24:56.722+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2025-02-21T17:27:21.936+0000] {processor.py:186} INFO - Started process (PID=23074) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:27:21.938+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-21T17:27:21.940+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:27:21.939+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:27:21.942+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:27:21.942+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-21T17:27:21.942+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:27:21.942+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-21T17:27:21.942+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:27:21.950+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2025-02-21T17:28:20.828+0000] {processor.py:186} INFO - Started process (PID=23573) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:28:20.830+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-21T17:28:20.833+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:28:20.832+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:28:20.835+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:28:20.834+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-21T17:28:20.835+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:28:20.835+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-21T17:28:20.835+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:28:20.843+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2025-02-21T17:29:17.991+0000] {processor.py:186} INFO - Started process (PID=24015) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:29:17.993+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-21T17:29:17.996+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:29:17.996+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:29:17.998+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:29:17.998+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-21T17:29:17.999+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:29:17.999+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-21T17:29:17.999+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:29:18.006+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.020 seconds
[2025-02-21T17:30:17.261+0000] {processor.py:186} INFO - Started process (PID=24723) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:30:17.264+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-21T17:30:17.266+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:30:17.265+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:30:17.268+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:30:17.268+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-21T17:30:17.268+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:30:17.268+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-21T17:30:17.268+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:30:17.275+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2025-02-21T17:31:12.889+0000] {processor.py:186} INFO - Started process (PID=25016) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:31:12.892+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-21T17:31:12.893+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:31:12.893+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:31:12.896+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:31:12.895+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-21T17:31:12.896+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:31:12.896+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-21T17:31:12.896+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:31:12.903+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2025-02-21T17:32:09.063+0000] {processor.py:186} INFO - Started process (PID=25264) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:32:09.065+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-21T17:32:09.066+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:32:09.066+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:32:09.069+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:32:09.068+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-21T17:32:09.069+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:32:09.069+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-21T17:32:09.069+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:32:09.076+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2025-02-21T17:33:06.207+0000] {processor.py:186} INFO - Started process (PID=25490) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:33:06.209+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-21T17:33:06.211+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:33:06.211+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:33:06.213+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:33:06.213+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-21T17:33:06.213+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:33:06.213+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-21T17:33:06.214+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:33:06.220+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2025-02-21T17:34:01.703+0000] {processor.py:186} INFO - Started process (PID=25713) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:34:01.705+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-21T17:34:01.707+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:34:01.707+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:34:01.710+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:34:01.709+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-21T17:34:01.710+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:34:01.710+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-21T17:34:01.710+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:34:01.718+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.020 seconds
[2025-02-21T17:34:57.991+0000] {processor.py:186} INFO - Started process (PID=25939) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:34:57.994+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-21T17:34:57.996+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:34:57.995+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:34:57.998+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:34:57.997+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-21T17:34:57.998+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:34:57.998+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-21T17:34:57.998+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:34:58.006+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2025-02-21T17:35:53.016+0000] {processor.py:186} INFO - Started process (PID=26153) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:35:53.019+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-21T17:35:53.021+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:35:53.020+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:35:53.023+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:35:53.023+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-21T17:35:53.024+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:35:53.023+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-21T17:35:53.024+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:35:53.032+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2025-02-21T17:36:48.980+0000] {processor.py:186} INFO - Started process (PID=26410) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:36:48.982+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-21T17:36:48.983+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:36:48.983+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:36:48.986+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:36:48.985+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-21T17:36:48.986+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:36:48.986+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-21T17:36:48.986+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:36:48.993+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2025-02-21T17:37:42.856+0000] {processor.py:186} INFO - Started process (PID=26674) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:37:42.859+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-21T17:37:42.860+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:37:42.860+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:37:42.863+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:37:42.862+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-21T17:37:42.863+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:37:42.863+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-21T17:37:42.863+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:37:42.870+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2025-02-21T17:38:37.320+0000] {processor.py:186} INFO - Started process (PID=26887) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:38:37.322+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-21T17:38:37.324+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:38:37.324+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:38:37.326+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:38:37.326+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-21T17:38:37.326+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:38:37.326+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-21T17:38:37.327+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:38:37.333+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2025-02-21T17:39:33.270+0000] {processor.py:186} INFO - Started process (PID=27114) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:39:33.272+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-21T17:39:33.274+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:39:33.274+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:39:33.277+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:39:33.276+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-21T17:39:33.277+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:39:33.277+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-21T17:39:33.277+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:39:33.285+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.020 seconds
[2025-02-21T17:40:29.090+0000] {processor.py:186} INFO - Started process (PID=27395) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:40:29.092+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-21T17:40:29.094+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:40:29.093+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:40:29.096+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:40:29.096+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-21T17:40:29.096+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:40:29.096+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-21T17:40:29.096+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:40:29.103+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2025-02-21T17:47:28.128+0000] {processor.py:186} INFO - Started process (PID=28898) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:47:28.131+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-21T17:47:28.132+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:47:28.132+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:47:28.135+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:47:28.135+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-21T17:47:28.135+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:47:28.135+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-21T17:47:28.135+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:47:28.142+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2025-02-21T17:48:25.885+0000] {processor.py:186} INFO - Started process (PID=29253) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:48:25.887+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-21T17:48:25.889+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:48:25.889+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:48:25.891+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:48:25.891+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-21T17:48:25.891+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:48:25.891+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-21T17:48:25.892+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:48:25.898+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2025-02-21T17:49:24.269+0000] {processor.py:186} INFO - Started process (PID=29609) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:49:24.271+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-21T17:49:24.273+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:49:24.273+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:49:24.275+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:49:24.275+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-21T17:49:24.276+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:49:24.276+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-21T17:49:24.276+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:49:24.283+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2025-02-21T17:50:23.938+0000] {processor.py:186} INFO - Started process (PID=29939) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:50:23.940+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-21T17:50:23.942+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:50:23.941+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:50:23.944+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:50:23.943+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-21T17:50:23.944+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:50:23.944+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-21T17:50:23.944+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:50:23.950+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2025-02-21T17:51:21.790+0000] {processor.py:186} INFO - Started process (PID=30292) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:51:21.792+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-21T17:51:21.794+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:51:21.794+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:51:21.797+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:51:21.796+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-21T17:51:21.797+0000] {logging_mixin.py:190} INFO - [2025-02-21T17:51:21.797+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-21T17:51:21.797+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-21T17:51:21.804+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.020 seconds
